### PR TITLE
Release navigating incubator

### PR DESCRIPTION
**Pull request by justinmclean**
First draft of a release of navigating ASF incubator slides

### CODE REVIEW

**Comment by chrisdutz** [comment]
Looks legit to me ... +1 for merging ...

**Comment by lfrancke** [comment]
Justin, do you know why all your PRs include those two commits:

Add MIT license to LICENSE. c539451
generate date and all people are cool 60fcb61

That's a bit weird, no?

**Comment by justinmclean** [comment]
No idea sorry. The file diff show no differences re those commits, possibly my master wasn't in sync with upstream when I made the PR??

File: content/ApacheWay/NavigatingASFIncubation/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+`./src/main/asciidoc/aboutme.adoc`
+
+Then build the slides to generate the HTML.
+
+## How to Build
+
+To installed the needed dependencies run:
+
+`install-deps.sh`
```
[the remark this answers is not in the document]
Comment (justinmclean) [reply content/ApacheWay/NavigatingASFIncubation/README.md]: Anything that makes like easier for people to install / setup his good to included I think. Added OSX bit

**Comment by soenkeliebau** [comment]
The build doesn't work for me on Fedora, Lars and I just looked at it and it seems that the use of <outputFile> in the asciidoctor plugin doesn't support modules that write multiple files.

Whether it works or not seems to be dependent on the order that the OS delivers files to the module, for me the output is:

`
[INFO] --- asciidoctor-maven-plugin:2.0.0-RC.1:process-asciidoc (generate-slides) @ navigating-ASF-incubation ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 0 resource
[INFO] Rendered /home/sliebau/Downloads/apache-training-navigating-ASF-incubation-1.0-incubating/src/main/asciidoc/index.adoc
[WARNING] Duplicated destination found: overwriting file: /home/sliebau/Downloads/apache-training-navigating-ASF-incubation-1.0-incubating/target/generated-slides/index.html
[INFO] Rendered /home/sliebau/Downloads/apache-training-navigating-ASF-incubation-1.0-incubating/src/main/asciidoc/projectstats.adoc
[WARNING] Duplicated destination found: overwriting file: /home/sliebau/Downloads/apache-training-navigating-ASF-incubation-1.0-incubating/target/generated-slides/index.html
[INFO] Rendered /home/sliebau/Downloads/apache-training-navigating-ASF-incubation-1.0-incubating/src/main/asciidoc/aboutme.adoc
[INFO] 
`

So the only slide I get is the about me slide, as it is rendered last and overwrites all other content. For Lars on Mac, index.adoc is the last file rendered, so for him it works.

**Comment by lfrancke** [comment]
In short: We have to find a different way than "outputFile" to generate the index.html file. Our idea was to just create it as is and then rename later in the Maven process.

We also need to fix this in the template.

**Comment by lfrancke** [comment]
While debugging this I also found the reference to Chris' home directory in the POM is still in there

`svgbob>/Users/christofer.dutz/.cargo/bin/svgbob</svgbob>`

I would suggest another cleanup round on the template and then apply it here.
It would also be great if we could create a parent POM so that we don't have to make improvements to the POM in all the subprojects.

**Comment by justinmclean** [comment]
given the outputFile issue been solved in another PR. I assume this is good to merge?

**Comment by soenkeliebau** [comment]
LGTM

**Pull request merged**